src/adapters/plugin/adapter-manager-proxy.js

```diff
@@ -99,6 +102,10 @@ class AdapterManagerProxy extends EventEmitter {
         this.pluginClient.sendNotification(Constants.ADAPTER_UNLOADED, {
           adapterId: adapter.id,
         });
+        return;
+
+      case Constants.UNLOAD_PLUGIN:
```

Comment:
Can you add this message to the Adapter IPC documentation in the wiki?

Reply:
Yep - was planning on doing that.